deployments/arbitrum/usdc.n/configuration.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "Compound USDC(native)",
+  "symbol": "cUSDCNv3",
+  "baseToken": "USDC",
+  "baseTokenAddress": "0xaf88d065e77c8cC2239327C5EDb3A432268e5831",
+  "baseTokenPriceFeed": "0x50834F3163758fcC1Df9973b6e91f0F0F0434aD3",
+  "borrowMin": "100e6",
```

Comment:
We can probably crank down the `borrowMin`. Maybe `10e6`?

Reply:
👍  sounds good.